core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -523,7 +523,8 @@ export class SettingsTabComponent
     }).result.then(() => {
       this.editableExplorationBackendApiService.deleteExplorationAsync(
         this.explorationId).then(() => {
-        this.windowRef.nativeWindow.location = this.CREATOR_DASHBOARD_PAGE_URL;
+        this.windowRef.nativeWindow.location = (
+          this.CREATOR_DASHBOARD_PAGE_URL as unknown as Location);
```

Comment:
Why `unknown`?

Reply:
Can `new Location(this.CREATOR_DASHBOARD_PAGE_URL)` be used?

Reply:
Or something similar to construct the `Location` object.

Reply:
Why isn't it working, please add a TODO if you don't want to bother with this in this PR.